tools/ci_build/github/azure-pipelines/templates/jar-maven-signing-linux.yml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  - name: JarFileDirectory
+    type: string
+
+  - name: WorkingDirectory
+    type: string
+
+  - name: EnableJarMavenSigning
+    type: boolean
+    default: false
+
+steps:
+  - task: AzureKeyVault@2
+    displayName: 'Get GnuPG signing keys'
+    condition: and(succeeded(), eq('${{ parameters.EnableJarMavenSigning }}', true))
+    inputs:
+      azureSubscription: 'OnnxrunTimeCodeSign_20240611'
+      KeyVaultName: 'ort-release'
+      SecretsFilter: 'java-pgp-pwd,java-pgp-key'
+      RunAsPreJob: false
+
+  - task: CmdLine@2
+    displayName: 'Sign jar files: GnuPG and sha256'
+    condition: and(succeeded(), eq('${{ parameters.EnableJarMavenSigning }}', true))
+    inputs:
+      workingDirectory: '$(Build.BinariesDirectory)'
+      script: |
+        #!/bin/bash
+
```

Comment:
Add
```
set -e -x
```
to the beginning of the script.  Otherwise if any of the command failed we wouldn't know.

Reply:
You should not add e there.